packages/react-components/react-nav-preview/src/components/NavCategory/NavCategory.types.ts

```diff
@@ -1,17 +1,23 @@
-import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
-
-export type NavCategorySlots = {
-  root: Slot<'div'>;
-};
+import * as React from 'react';
+import { NavItemValue } from '../NavContext.types';
+import { NavCategoryContextValue } from '../NavCategoryContext';
 
 /**
  * NavCategory Props
  */
-export type NavCategoryProps = ComponentProps<NavCategorySlots> & {};
```

Comment:
Checking that you wanted to remove ComponentProps?  For example, Menu doesn't have slots, but it uses ComponentSlots with an empty type.  I'm not sure how not using ComponentProps will affect getSlots during rendering.

Reply:
Yep, was just stripping out anything unused, added back :-)